README.md

```diff
@@ -555,7 +555,11 @@ Front. Neurosci. (2017), 11:350.
 - Yousefzadeh, A., Masquelier, T., Serrano-Gotarredona, T., Linares-Barranco, B.,  
 *[Live demonstration: Hardware implementation of convolutional STDP for on-line visual feature learning](https://doi.org/10.1109/ISCAS.2017.8050395)*,  
 IEEE Int. Symp. Circuits and Systems (ISCAS), 2017.
-
+- <a name="Sironi18cvpr"></a>Sironi, A., Brambilla, M., Bourdis, N., Lagorce, X., Benosman, R.,    
+*[HATS: Histograms of Averaged Time Surfaces for Robust Event-based Object Classification](https://arxiv.org/abs/1803.07913)*,  
+IEEE Conf. Computer Vision and Pattern Recognition (CVPR) 2018. 
+    - [N-CARS Dataset](http://www.prophesee.ai/dataset-n-cars/): A large real-world event-based dataset for car classification.
```

Comment:
It would be nice if this entry has a link to the dataset below, instead of having two equal links on the same page.

Reply:
sure ! Thanks